internal/backend/remote-state/azure/backend_state.go

```diff
@@ -92,7 +92,7 @@ func (b *Backend) StateMgr(ctx context.Context, name string) (statemgr.Full, err
 	stateMgr := remote.NewState(client, b.encryption)
 
 	// Grab the value
-	if err := stateMgr.RefreshState(); err != nil {
+	if err := stateMgr.RefreshState(context.TODO()); err != nil {
```

Comment:
To make it easier to update down the line, would it make sense to declare a `ctx := context.TODO()` at the top of the function?

Reply:
My goal was to change the TODO callers as little as possible and keep this totally systematic so it's easier to review this. 

Depending on what we decide to do about the requirement to disable cancellation on the contexts here we might end up reorganizing this code in a more significant way later anyway, so I don't want to try to guess what that future shape might look like right now, but I can change it as you suggested if you feel strongly about it.